gitman/models/config.py

```diff
@@ -180,11 +180,38 @@ def run_scripts(self, *names, depth=None, force=False, show_shell_stdout=False):
 
         return count
 
+    @classmethod
+    def split_name_and_rev(cls, name_rev):
+        true_name = name_rev
+        rev = None
+        if "@" in name_rev:
+            name_split = name_rev.split("@")
+            true_name = name_split[0]
+            rev = name_split[1]
+        return true_name, rev
+
+    @classmethod
+    def remap_names_and_revs(cls, names):
```

Comment:
If possible, I think these should be private methods.

Reply:
By private you mean adding `__` to the beginning of the method name?

Reply:
Just a single `_` to indicate to clients that these methods aren't meant for public consumption and might change.